pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2894,3 +2933,49 @@ Suggestion.ActivityInstance:
       description: Activity Instance Name, rendered, as could be seen on the dashboard
       type: string
       example: "Medical Record Release Form #2"
+DateRange:
+  type: object
+  required:
+    - from
+    - to
+  properties:
+    from:
+      type: string
+      description: the lower constraint of the range. This value is inclusive. The date format is a full, extended ISO-8601 Calendar date and time in the format `YYYY-MM-DDTHH:MM:SSZ` in the UTC time zone.
+      nullable: true
+      format: date-time
+    to:
+      type: string
+      description: the upper constraint of the range. This value is inclusive. The date format is a full, extended ISO-8601 Calendar date and time in the format `YYYY-MM-DDTHH:MM:SSZ` in the UTC time zone.
+      nullable: true
+      format: date-time
+IntegerRange:
+  type: object
+  required:
+    - from
+    - to
+  properties:
+    from:
+      type: integer
+      description: the lower constraint of the range. This value is inclusive.
+      nullable: true
+      format: int64
+    to:
+      type: integer
+      description: the upper constraint of the range. This value is inclusive.
+      nullable: true
+      format: int64
+Sorting:
+  type: object
+  properties:
+    column:
+      description: the column name
+      nullable: true
+      type: string
```

Comment:
This description might need to be more specific. the value of `column` has to be one of the attribute names returned with each audit trail record right? (e.g., studyGuid, operatorGuid, etc from your AuditTrailResponse definition).

Reply:
Right. I was thinking the we might define enum (mapping between column name and database column name) or just get the DB column name using reflection mechanism.